File: historyscan-back/src/main/java/io/sch/historyscan/infrastructure/features/codebase/clone/CodeBaseToAddDTO.java
package io.sch.historyscan.infrastructure.features.codebase.clone;

import com.fasterxml.jackson.databind.annotation.JsonSerialize;

@JsonSerialize
public record CodeBaseToAddDTO(String url,
                               String publicKey,
                               String name,
                               String branch
) {
}

File: historyscan-back/src/main/kotlin/io/sch/historyscan/infrastructure/features/codebase/clone/CodeBaseToAddDTO.kt
package io.sch.historyscan.infrastructure.features.codebase.clone

import com.fasterxml.jackson.databind.annotation.JsonSerialize

@JsonSerialize
class CodeBaseToAddDTO(
    val url: String,
    val publicKey: String,
    val name: String,
    val branch: String
) {}
